README-ENGINES.md

```diff
@@ -53,7 +53,7 @@ user/admin the set of commands and parameter types a given ENGINE
 implementation supports, and for an application to directly feed string
 based input to those ENGINEs, in the form of name-value pairs. This is an
 extensible way for ENGINEs to define their own "configuration" mechanisms
-that are specific to a given ENGINE (eg. for a particular hardware
+that are specific to a given ENGINE (e.g. for a particular hardware
```

Comment:
Personally I prefer `e.g.`, as done here; however note often that using `eg.` can be a matter of style